Topic4Services/src/payments/CashPayment.java

```diff
@@ -0,0 +1,24 @@
+package payments;
+
+import discounts.Discount;
+import discounts.NinetyPercentMostExpensiveDiscount;
+
+public class CashPayment implements Payment {
+
+	private Discount discount;
+
+	public CashPayment() {
+		discount = new NinetyPercentMostExpensiveDiscount();
+	}
+
+	@Override
+	public void pay(float total) {
+

```

Comment:
I remember that you mentioned that the pay methods will be emty, but here you could add a TODO comment like "To be defined in the final project"